sdk/webpubsub/azure-resourcemanager-webpubsub/src/test/java/com/azure/resourcemanager/webpubsub/WebPubSubManagerTests.java

```diff
@@ -35,7 +34,7 @@ public class WebPubSubManagerTests extends TestBase {
 
     @Override
     public void beforeTest() {
-        final TokenCredential credential = new DefaultAzureCredentialBuilder().build();
+        final TokenCredential credential = new AzurePowerShellCredentialBuilder().build();
```

Comment:
Can the live test always get `AzurePowerShellCredential`? I thought Powershell credential can only be achieved on windows machine. And CI can run on Linux? Correct me if I'm wrong.

Reply:
1. This fix is ​​only for live tests, the detail infomation following as : https://github.com/Azure/azure-sdk-for-java-pr/issues/1419#issuecomment-2193003772.
2. Both PowerShell and Azure CLI can be install and run on Linux.

Reply:
Most of it is from EngSys and Arch (Srikanta etc.), so it should work as such.

We had mgmt test in datafactory that didn't do the filter, and it works for all platforms https://github.com/Azure/azure-sdk-for-java/pull/40861

(the reason we do filter is to avoid consuming too many Azure resources)